### DISCUSSION TITLE
[Help Wanted] Which function can zoom in the size of a font?

### DISCUSSION THREAD

**Discussion by moleoxa** [fonttools/fonttools#3711] [Q&A] [answered]
The default size of a font seems small for me. I always need 120% zoom in while use the browser. But Microsoft Yahei is 100% OK.

What I want to do: Enlarge every character to 1.1x until it behave like Microsoft Yahei (Windows Default)

Anybody can tell me whcih command/function is I need? Or is there an useful keyword for me to search at user manual docs?

**Comment by behdad** [comment, marked as answer]
You need to do something like:
```
font = TTFont("myfont.ttf")
font['head'].unitsPerEm /= 1.2
font.save("scaled.ttf")
```
